dev/NavigationView/NavigationView.cpp

```diff
@@ -1544,12 +1536,12 @@ void NavigationView::UpdatePaneLayout()
                                 winrt::VisualStateManager::GoToState(*this, c_separatorCollapsedStateName, false);
                                 return 0.0;
                             }
-                            else if (totalAvailableHeight > menuItemsActualHeight + footersActualHeight)
+                            else if (totalAvailableHeight > menuItemsActualHeight + footerGroupActualHeight)
```

Comment:
In that MUXControlsTestPage first NavigationView test page, totalAvailableHeight happens to be menuItemsActualHeight + footerGroupActualHeight + 4.0 

The 4.0 comes from the missing paneFooter.Margin.Bottom mentioned above. Once that is fixed, you will end up with  
totalAvailableHeight == menuItemsActualHeight + footerGroupActualHeight and this branch will incorrectly be skipped.

I believe this is because instead of:
else if (totalAvailableHeight > menuItemsActualHeight + footerGroupActualHeight)
we should be using:
else if (totalAvailableHeight >= menuItems.DesiredSize.Height + footerGroupActualHeight)

Reply:
I don't think this is being incorrectly skipped. If `totalAvailableHeight == menuItemsActualHeight + footerGroupActualHeight`, logically, there should be a pane separator to distinguish the two sections rather than it being collapsed.

Reply:
Discussed offline. You're right, it would make more sense to add the >=